socs/agents/ucsc_radiometer/agent.py

```diff
@@ -95,6 +91,9 @@ def acq(self, session, params=None):
 
             if params['test_mode']:
                 break
+            else:
+                pm = Pacemaker(1 / 60, quantize=False)
```

Comment:
I don't think we want to be recreating the `Pacemaker` object every loop, since that wipes out its internal tracking of when the next sample time should be. I _think_ this should work if the `Pacemaker` instantiation is still up at the start of the Process and just `pm.sleep()` is in this `else` block.

If that doesn't work, can you confirm that the sample rate is still as specified? My preference would still be to avoid this construction by Mocking out sleep in the tests.